tests/requirements.txt

```
@@ -1,7 +1,7 @@
 Sphinx
 WebTest
 colander>=1.0b1
-marshmallow>=2.0.0,<3.0.0
+marshmallow>=2.0.0
```

```suggestion
marshmallow
```

---

Well, you probably don't want to support marshmallow 1, right? 😃 

I'd recommend

```suggestion
marshmallow>=2.0.0,4.0.0
```

---

If you don't specify a version it will take the latest one. However you don't know yet if 4.0 will be supported or not, so it is not the responsibility of this lib to decide.

More information about how to handle libraries dependencies in Python are available there: https://github.com/mozilla-services/servicedenuages.fr/blob/master/content/2017.02.handling-python-projects-dependencies.rst

---

`marshmallow>=2.0.0` seems appropriate